types/query.d.ts

```diff
@@ -99,82 +99,10 @@ declare module 'mongoose' {
     PopulateOption,
     SessionOption {
     arrayFilters?: { [key: string]: any }[];
-    batchSize?: number;
```

Comment:
What makes you think that deleting all the query options that operations other than `updateOne()` use is a valid solution?